### ISSUE TITLE
Allow Discussion default value is "None"

### ISSUE THREAD

**Issue by jtmolon** [plone/plone.app.dexterity#195]
Why is the default value of allow_discussion[1] 'None' in the Allow Discussion behavior?

I think that if Discussion is allowed for a content type, the default for the instances of this type should be 'True'.

Cheers.

[1] https://github.com/plone/plone.app.dexterity/blob/master/plone/app/dexterity/behaviors/discussion.py#L34

**Comment by tisto** [comment]
If allow_discussion would be true by default it would be impossible to override by the content type specific setting. Please feel free to re-open if you disagree.

**Comment by gforcada** [comment]
Maybe renaming (visually) to *inherit* would make more sense?

**Comment by tisto** [comment]
@gforcada yes. I guess it would make sense to change the label and add a description.

**Comment by pbauer** [comment]
I once wrote down the whole story about the settings in https://github.com/plone/Products.CMFPlone/issues/972

**Comment by davilima6** [comment]
:+1: for "Inherit from parent" or "Inherit the value from parent", as suggested in #944, in addition to all the suggestions from #972

**Comment by jtmolon** [comment]
Now I understand. The proposed changes would definitely make it clearer.

**Comment by lccruz** [comment]
+1

**Comment by zupo** [comment]
+1 for proposed change

**Comment by jensens** [comment]
Closing as stale. allow_discussion default is None. By design (inherit global setting), not a bug.
If you think this issue is still relevant please reopen it. Thank you for the original report!
